src/shared/modules.mjs

```diff
@@ -854,6 +855,137 @@ const generateEventListenResponse = json => {
     return json
 }
 
+const getAnyOfSchema = (inType, json) => {
```

Comment:
Just making sure that I understand...

This method localizes the schema, and then reconstructs the anyOf $ref links.

It does this to ensure that we have a simple version of the schema, w/ links, instead of a more complicated one with complicate/multiple links?

Reply:
yes, here we are populating content of the anyOf types reference. 
in case NavigationIntent, first we make it as localised, to get the solved $ref links. Then only we will able to identify it is anyOf or not (in case of $ref). Since localise resolve all subSchemas, we just check higher schema is $ref or not, if then get its definitions . i.e, 
"anyOf": [ { "$ref": "#/x-schemas/Intents/HomeIntent"},  
                  { "$ref": "#/x-schemas/Intents/LaunchIntent"},
                  { "$ref": "#/x-schemas/Intents/EntityIntent" },
                  { "$ref": "#/x-schemas/Intents/PlaybackIntent"},
                  { "$ref": "#/x-schemas/Intents/SearchIntent"},
                  { "$ref": "#/x-schemas/Intents/TuneIntent"},
                ]